CONTRIBUTING.md

```diff
@@ -126,6 +126,19 @@ branch that is checked out in a separate worktree.
     git log --stat -p aaaaaa...bbbbbb
     ```
 
+## Polish a change
+
+Prior to pushing your change, remember to run clang-format on the `c/cpp` files, clang-format workflow may fail otherwise. You can do so by running the following command.
```

Comment:
Try linking as much as possible; you never know whether the context might be valuable to a user. Here I would link `clang-format` to the clang-format website and `clang-format workflow` to our clang-format workflow file.